manifests/cf-manifest/spec/support/manifest_helpers.rb

```diff
@@ -68,20 +101,30 @@ def workdir
     Cache.instance.workdir ||= $workdir
   end
 
+  def manifest_without_vars_store
+    Cache.instance.manifest_without_vars_store ||= \
```

Comment:
Originally this used ```Cache.instance.manifest_with_defaults``` which caused the test to use the wrong manifest.
I added a separate variable.